### ISSUE TITLE
$schema disappears when deserializes

### ISSUE THREAD

**Issue by keighrim** [lapps/org.lappsgrid.metadata#17]
`$schema` field disappears when the following code runs: 

``` java 
String json = service.getMetadata();
Data data = Serializer.parse(json, Data.class); // deserialize data
ServiceMetadata metadata = new ServiceMetadata((Map) data.getPayload()); // deserialize payload
```

**Comment by keighrim** [comment]
Possibly related [this part](https://github.com/lapps/org.lappsgrid.metadata/blob/220e3747bd1c0df679a9879c54be2319d0aca384/src/main/groovy/org/lappsgrid/metadata/ServiceMetadata.groovy#L136) of code, as `((Map) data.getPayload()).keySet()` returns 
```
[$schema, name, version, description, vendor, license, requires, produces]
```

(I'm using 1.4.0-SNAPSHOT)

**Comment by keighrim** [comment]
Addressed in #18. Closing.
